store/cassandra/cassandra.go

```diff
@@ -109,6 +109,22 @@ func PrepareChunkData(span uint32, data []byte) []byte {
 	return buf.Bytes()
 }
 
+// the timeout value used to be an integer specifying the number of milliseconds
+// we want to convert it to a duration string, but we need to stay backwards-compatible for now
+func ConvertTimeout(timeout string) time.Duration {
+	if timeoutI, err := strconv.Atoi(timeout); err == nil {
+		log.Warn("cassandra_store: specifying the timeout as integer is deprecated, please use a duration value")
+		return time.Duration(timeoutI) * time.Millisecond
+	} else {
+		timeoutD, err := time.ParseDuration(timeout)
+		if err == nil {
+			return timeoutD
+		}
+		log.Warn("cassandra_store: invalid duration value %s, assuming default (1s)", timeout)
```

Comment:
if the input can't be parsed as an int, nor as a duration specifier, we should error and abort.
just like we handle all other invalid configuration

Reply:
Changed that